### ISSUE TITLE
[Docs Feedback]: Improve operational instructions for Gitpod update

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/website#2721]
1. Share `Live update (No downtime, higher risk, low cost)` for Single Cluster Ref Arch is not supported.
2. Share `maintenance window` must start by checking to assert that there are no running workspaces. This is a :skateboard: , for the future, we'll want a means to "enable" maintenance windows via webapp or perhaps using `gpctl` to temporarily disable routing of start workspace requests.

### Page URL

https://www.gitpod.io/docs/self-hosted/latest/updating#live-update-no-downtime-higher-risk-low-cost 
https://www.gitpod.io/docs/self-hosted/latest/updating#maintenance-window-higher-downtime-low-risk-and-medium-cost---recommended

### User feedback

**Live update**
If a user upgrades a cluster while workspaces are live, and the ws-daemon restarts while a workspace is stopping (backing up) they could encounter data loss. This is why this approach is considered high risk, and why a maintenance window is required. In other words, Single Cluster Ref Arch in its current form does not support "No downtime" upgrades.

In other words :point_up: is not an edge case when workspaces are running. The more workspaces running during the upgrade, the greater there is a chance of data loss, and having to do related clean-up of the workspaces.

**Maintenance window**
This page simply needs to be updated to inclube verbiage that indicates the admins must check for running workspaces (imagebuild, prebuild, and regular) before starting the upgrade.

### Definition of done

The documentation is updated to clearly outline that No download upgrades for Single Cluster Ref Arch is not supported. The related PR must be approved by the CX and TS teams.

**Comment by kylos101** [comment]
@jldec hey, are there any plans for webapp to include the capability to enable maintenance windows, and avoid sending start workspace requests for imagebuild/prebuild/regular? If so, can you share roughly when after UBP?
